templates/projects/_helpers.html

```diff
@@ -325,11 +325,11 @@ <h2><strong> {{_('Overall progress')}}</strong>: {{ overall_progress | round | i
     <div class="btn-group btn-contribute">
         {% if current_user.admin or (current_user.subadmin and current_user.id in project.owners_ids) %}
         <div id="btn-contribute-now" class="btn-group btn-contribute" role="group">
-              <a class="btn btn-lg btn-primary btn-padding" role="button" href="{{url_for('project.presenter', short_name=project.short_name)}}">{{ _('Start Contributing Now!') }}</a>
+              <a id="new_task_admin" class="btn btn-lg btn-primary btn-padding" role="button" href="{{url_for('project.presenter', short_name=project.short_name)}}">{{ _('Start Contributing Now!') }}</a>
```

Comment:
as this will be available for admin and subadmin coowners, should this be named accordingly? ex. `new_task_admin_subadmin_coowners` or similar

Reply:
Makes sense. But since the "if" condition could change in the future, I would prefer the name of "id" to be boarder and conciser.

Reply:
Let's go with the current context and as need arise with change in scope we can updates names relatively in future.  This is inline with similar examples at other places in the code, where names are defined as per current scope. Thanks!

Reply:
This doesn't seem to be a good practice to me. I think the tag id "new_task_admin" should be good enough to distinguish another tag id "new_task_user".

Reply:
do you agree that this is not just for admin, but admin and subadmin coowner?